Update kubectl subresource documentation (#49824)
kubernetes/website: content/en/docs/reference/kubectl/conventions.md

## Subresources

> * You can use the `--subresource` flag for kubectl commands like `get`, `patch`,

Reviewer: (nit)
```suggestion
* You can use the `--subresource` argument for kubectl subcommands such as `get`, `patch`,
```

`edit`, `apply` and `replace` to fetch and update subresources for all resources that

> support them. Currently, only the `status` and `scale` subresources are supported.

Reviewer: ```suggestion
support them. In Kubernetes version {{< skew currentVersion >}}, only the `status` and `scale` subresources are supported.
```
